### DISCUSSION TITLE
Support for SVT-AV1-PSY

### DISCUSSION THREAD

**Discussion by kuzon3** [staxrip/staxrip#1302] [Ideas]
Any chance of adding support for SVT-AV1-PSY by gianni-rosato?

**Comment by Dendraspis** [comment]
It was already considered, but after many, many hours it was not able to get a properly working executable out of it. So we're still waiting, maybe with their next release.

**Reply by kuzon3** [reply]
Awesome. ATM I'm using there 2.0.0 with staxrip and everything is going good 👍

**Comment by AdamBorealis** [comment]
I'm seeing patch notes saying SVT-AV1-PSY support, but I'm not noticing any changes on the GUI.  Does that mean I have to compile the SvtAv1EncAppSVT.exe using the PSY branch / replace the .exe in the staxrip->Encoders->SvtAv1EncApp folder?

If so, that's fine, but it's not spelled out very clearly from the patch notes.

**Reply by Dendraspis** [reply]
Exactly, whereas you can use the compiled build(s) instead.

Currently there is support for it, but it's not shipped with `v.2.38` because there were still some issues with the official PSY version. I guess `v2.39` will come with the PSY version.
